multicluster/config/samples/clusterset_init/multicluster_membercluster_template.yaml

```diff
@@ -23,6 +23,5 @@ spec:
       secret: leader-access-token
       server: https://<LEADER_CLUSTER_IP>:6443
   members:
-    - clusterID: test-cluster-leader
```

Comment:
@luolanzone : is it a bug earlier? Or maybe `members` is useless in a member cluster ClusterSet? Should we remove it then?

Reply:
I removed `members` from all member cluster yamls.